website/versioned_docs/version-22.4/CLI.md

```diff
@@ -0,0 +1,347 @@
+```
+
+Run tests related to `path/to/fileA.js` and `path/to/fileB.js`:
+
+```bash
+jest --findRelatedTests path/to/fileA.js path/to/fileB.js
+```
+
+Run tests that match this spec name (match against the name in `describe` or
+`test`, basically).
+
+```bash
+jest -t name-of-spec
+```
+
+Run watch mode:
+
+```bash
+jest --watch #runs jest -o by default
+jest --watchAll #runs all tests
+```
+
+Watch mode also enables to specify the name or path to a file to focus on a
+specific set of tests.
+
+## Using with yarn
+
+If you run Jest via `yarn test`, you can pass the command line arguments
+directly as Jest arguments.
+
+Instead of:
+
+```bash
+jest -u -t="ColorPicker"
+```
+
+you can use:
+
+```bash
+yarn test -u -t="ColorPicker"
+```
+
+## Using with npm scripts
+
+If you run Jest via `npm test`, you can still use the command line arguments by
+inserting a `--` between `npm test` and the Jest arguments.
+
+Instead of:
+
+```bash
+jest -u -t="ColorPicker"
+```
+
+you can use:
+
+```bash
+npm test -- -u -t="ColorPicker"
+```
+
+## Options
+
+_Note: CLI options take precedence over values from the
+[Configuration](Configuration.md)._
+
+<AUTOGENERATED_TABLE_OF_CONTENTS>
+
+---
+
+## Reference
+
+### `jest <regexForTestFiles>`
+
+When you run `jest` with an argument, that argument is treated as a regular
+expression to match against files in your project. It is possible to run test
+suites by providing a pattern. Only the files that the pattern matches will be
+picked up and executed. Depending on your terminal, you may need to quote this
+argument: `jest "my.*(complex)?pattern"`. On Windows, you will need to use `/`
+as a path separator or escape `\` as `\\`.
+
+### `--bail`
+
+Alias: `-b`. Exit the test suite immediately upon the first failing test suite.
+
+### `--cache`
+
+Whether to use the cache. Defaults to true. Disable the cache using
+`--no-cache`. _Note: the cache should only be disabled if you are experiencing
+caching related problems. On average, disabling the cache makes Jest at least
+two times slower._
+
+If you want to inspect the cache, use `--showConfig` and look at the
+`cacheDirectory` value. If you need to clear the cache, use `--clearCache`.
+
+### `--changedFilesWithAncestor`
+
+Runs tests related to the current changes and the changes made in the last
+commit. Behaves similarly to `--onlyChanged`.
+
+### `--changedSince`
+
+##### available in Jest **22.2.0+**
+
+Runs tests related the changes since the provided branch. If the current branch
+has diverged from the given branch, then only changes made locally will be
+tested. Behaves similarly to `--onlyChanged`.
+
+### `--ci`
+
+When this option is provided, Jest will assume it is running in a CI
+environment. This changes the behavior when a new snapshot is encountered.
+Instead of the regular behavior of storing a new snapshot automatically, it will
+fail the test and require Jest to be run with `--updateSnapshot`.
+
+### `--clearCache`
+
+##### available in Jest **22.0.0+**
```

Comment:
These should not really be necessary anymore. Not a change this PR needs to make, though 🙂